cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, ontology=None):
+        """Return a display name for the given variable.
+
+        Looks for annotation in the ontology first, then cmeta:id if present, or the name attribute if not
```

Comment:
```suggestion
        Looks for annotation in the ontology first (if given), then cmeta:id if present, or the name attribute if not
```

Reply:
Wait @jonc125 is this what we want? Would we not prefer it to return (1) a specific ontology if given, else an annotation from _any_ ontology (2) if 1 fails, then a cmeta:id, (3) a name attribute